server/db/seed.js

```diff
@@ -78,13 +82,15 @@ async function seed() {
       conversationId: hualingConvo.id,
       senderId: hualing.id,
       text: "a test message",
+      isRead: false,
     });
   }
 
   await Message.create({
     conversationId: hualingConvo.id,
     senderId: hualing.id,
     text: "😂 😂 😂",
+    isRead: false,
   });
```

Comment:
You can also use defaultValue: false when setting up the database model.

Reply:
Thanks for the suggestion.